### ISSUE TITLE
Categorized Products in Webpage

### ISSUE THREAD

**Issue by JacobYim** [JacobYim/CSE442-BuyNSell#3]
How classified the products in Webpage?

It is significant issue for the database, datascience, and web design.

How about the

**Comment by JacobYim** [comment]
![image](https://user-images.githubusercontent.com/25061080/52888482-cc366000-3149-11e9-9aa4-4bc0f7ac4e71.png)

how about this? This is amazon categories for their products.

**Comment by ChoongLiang** [comment]
Yeah, what Amazon has created should be complete and sufficient as a base prototype.
